### PR TITLE
Update configuration to run the operator on OC4

### PR DESCRIPTION
**Pull request by soufianebenali**
We are updating the configuration to run the operator on OpenShift4

### CODE REVIEW

File: auth0.md [hunk 1]
```diff
@@ -81,4 +81,4 @@ M2M allows to work with auth0 Management API from backend: fetch oauth providers
 Everything is ready for work:
 
 - Take M2M **Client Secret** and **Client ID** and place it to backend ENV variable
-- Take SPA **Client ID** and place to aut0 js sdk.
\ No newline at end of file
+- Take SPA **Client ID** and place to aut0 js sdk.
```
Comment (sebgoa) [inline auth0.md]: auth0 ? missing an h ?

**Comment by sebgoa** [comment]
lgtm for me

@soufianebenali let us know when this PR is not WIP anymore

File: README.md [hunk 2]
```diff
@@ -32,6 +32,15 @@ kubectl create -f https://raw.githubusercontent.com/triggermesh/operator/master/
 
 Deploy the Operator in your k8s cluster:
 
+# Prerequisite
+
+Install the following operators from RedHat OperatorHub:
+
+```
+Knative Eventing Operator
+OpenShift Serverless Operator
```
[the remark this answers is not in the document]
Comment (soufianebenali) [reply README.md]: maybe something like this https://<console-url>/k8s/ns/openshift-kube-apiserver-operator/operators.coreos.com~v1alpha1~ClusterServiceVersion/serverless-operator.v1.4.0

[the remark this answers is not in the document]
Comment (sebgoa) [reply README.md]: actually, is it this one: https://operatorhub.io/operator/knative-eventing-operator

[the remark this answers is not in the document]
Comment (soufianebenali) [reply README.md]: Yes but for Tekton I don't see any link

File: helm-charts/triggermesh/templates/backend/cluster-role-user.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      - githubsources
+      - gitlabsources
+      - inmemorychannels
+      - kafkasources
+      - kinesischannels
+      - kuberneteseventsources
+      - pipelineresources
+      - pipelineruns
+      - pipelines
+      - pods/exec
```
Comment (tzununbekov) [inline helm-charts/triggermesh/templates/backend/cluster-role-user.yaml]:
```
  - pods
  - pods/exec
  - pods/log
```
can we add these resources to user role?

Comment (srueg) [reply helm-charts/triggermesh/templates/backend/cluster-role-user.yaml]: Done

**Pull request merged**